SecurityService/Startup.cs

```diff
@@ -212,6 +224,28 @@ public void ConfigureContainer(ServiceRegistry services)
 
         private void ConfigureMiddlewareServices(IServiceCollection services)
         {
+            services.AddHealthChecks()
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("PersistedGrantDbContext"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Persisted Grant DB",
+                                  failureStatus:HealthStatus.Unhealthy,
+                                  tags:new string[] {"db", "sql", "sqlserver", "persistedgrant"})
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("ConfigurationDbContext"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Configuration DB",
+                                  failureStatus:HealthStatus.Unhealthy,
+                                  tags:new string[] {"db", "sql", "sqlserver", "configuration"})
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6239480)